content/article/my-perl-wishlist-invariant-sigils-part-1.md

```diff
@@ -0,0 +1,181 @@
+
+  {
+    "title"       : "My Perl Wishlist: Invariant Sigils (Part 1)",
+    "authors"     : ["christopher-white"],
+    "date"        : "2019-10-15T08:33:34",
+    "tags"        : ["perl-internals", "syntax", "internals",
+                    "compiler", "perl-5", "language", "language-design",
+                    "sigil", "parser", "parsing", "perl-parsing", "core", "c"],
+    "draft"       : true,
+    "image"       : "",
+    "thumbnail"   : "",
+    "description" : "The change I would most like to see in Perl 5, and why it might not happen",
+    "categories"  : "programming-languages"
+  }
+
+Pop quiz!  Q: What was my mistake in this line?
+
+```perl
+is(%HASH{answer}, 'forty-two', q[%HASH properly filled]);
```

Comment:
How about simplifying this a bit to:

    is %HASH{answer}, 'forty-two', '%HASH properly filled';